### PR TITLE
[zh] fix typo

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/47654/files

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/controllers/job.md [hunk 1]
```diff
@@ -1874,7 +1874,7 @@ and `.spec.completions` together such that `.spec.parallelism == .spec.completio
 When scaling down, Kubernetes removes the Pods with higher indexes.
 
 Use cases for elastic Indexed Jobs include batch workloads which require 
-scaling an indexed Job, such as MPI, Horovord, Ray, and PyTorch training jobs.
+scaling an indexed Job, such as MPI, Horovod, Ray, and PyTorch training jobs.
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/controllers/job.md:1877]: Line 1883 should be changed as well?

Comment (Arhell) [reply content/zh-cn/docs/concepts/workloads/controllers/job.md:1877]: Done, thanks

**Pull request merged**